scipy/stats/unuran/unuran_wrapper.pyi

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import TYPE_CHECKING, Union, Any, Callable, Tuple, Optional, List
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
+
+
+SeedType = Union[np.random.RandomState, np.random.Generator, int,
```

Comment:
Also some helpers are defined in `scipy._lib._util`.

Reply:
Changed. I now use the `SeedType` from `scipy._lib._util`